### DISCUSSION TITLE
Discussionsの「Close」とは「削除」のことですか？

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#472] [ヘルプ] [answered]

**Comment by kyntk** [comment, marked as answer]
@tsubolabo 

対象のDiscussionは削除しておりません。
おそらくDiscussionsがフィルタリングされていると思われます！
一覧から「Filter」をクリックして、「Closed」を選択するとCloseされたものも表示されます。

![image](https://user-images.githubusercontent.com/17251829/235443673-d481cdb9-70ab-4972-bbfe-f77b8409e9fa.png)

クローズについての運用方針はこちらのDiscussionで説明しておりますので、ご覧ください :pray:
https://github.com/increments/qiita-discussions/discussions/455

**Reply by kyntk** [reply]
@tsubolabo 
> あるいは、デフォルト設定を「Open」から「All」に変えていただく方が便利かも。

それができるといいのですが、Disucussions自体はGitHubの機能を使っているため、できることとできないことがあります。
詳しくはGitHubのドキュメントをご覧ください 🙇 
https://docs.github.com/ja/discussions

クローズのタイミングは管理上の都合と、運営の他にDiscussionsを立てた方ご自身でもクローズしていただけるものなので、現状のままとさせてください :pray:

**Reply by munierujp** [reply]
FYI: 私はこのリポジトリのすべてのアクティビティをウォッチしており、クローズなども含めメール通知で動向を追っています。

![018EBAC6-D59D-4930-BE70-0FDBA788E5B7](https://user-images.githubusercontent.com/20086673/235678564-e209424f-28be-4525-834d-7365a440cb90.png)

**Comment by kyntk** [comment]
回答済みのためCloseします

**Reply by tsubolabo** [reply]
>>あるいは、デフォルト設定を「Open」から「All」に変えていただく方が便利かも。

>それができるといいのですが、Disucussions自体はGitHubの機能を使っているため、できることとできないことがあります。

　今、気づきましたが、最近の Discussion 画面では Filter の既定値が Open から All に変わっているよう です。これなら、解決 即 Close でも問題なさそうですね。

**Reply by tanaka-asami415** [reply]
@tsubolabo 
本当ですね！教えて頂きありがとうございます！

**Reply by tsubolabo** [reply]
2024-08-08現在、またAllからOpenに戻っています。
これは GitHubが一方的に変えたものなのですか？　Qiitaの意思によって変えたものですか？

**Reply by tanaka-asami415** [reply]
@tsubolabo 
本当ですね…
こちらは、弊社で変更したものではなく、GitHub の仕様変更だと思われます🙇
